client/wildcard/src/hooks/useObservable.ts

```diff
@@ -28,6 +30,36 @@ export function useObservable<T>(observable: Observable<T>): T | undefined {
     return currentValue
 }
 
+/**
+ * React hook to get the latest value of an Observable.
+ *
+ * @description This is a slightly modified version of `useObservable` that returns a `LoadStatus` instead of `undefined` when the Observable hasn't emitted yet.
+ * @param observable The Observable to subscribe to.
+ * @returns [T | undefined, LoadStatus, Error | undefined]
+ */
+export function useEnhancedObservable<T>(observable: Observable<T>): [T | undefined, LoadStatus, any] {
```

Comment:
nit: suggestion for a more descriptive name

```suggestion
export function useObservableWithStatus<T>(observable: Observable<T>): [T | undefined, LoadStatus, any] {
```